lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -357,11 +357,9 @@ export abstract class ClientApplication {
                 proxyUrl: this.config.system.proxyUrl,
             },
             loggerOptions: {
-                logLevel: this.config.system.loggerOptions.logLevel,
-                loggerCallback: this.config.system.loggerOptions
-                    .loggerCallback,
-                piiLoggingEnabled: this.config.system.loggerOptions
-                    .piiLoggingEnabled,
+                logLevel: this.config.system.loggerOptions?.logLevel || LogLevel.Info,
```

Comment:
Like in browser, do this in `Configuration.ts`

Reply:
This error is occurring due to this bit in the Configuration.ts:-

`    auth,
    cache,
    system,
    telemetry
}: Configuration): NodeConfiguration {
    return {
        auth: { ...DEFAULT_AUTH_OPTIONS, ...auth },
        cache: { ...DEFAULT_CACHE_OPTIONS, ...cache },
        system: { ...DEFAULT_SYSTEM_OPTIONS, ...fixSystem },
        telemetry: { ...DEFAULT_TELEMETRY_OPTIONS, ...telemetry }
    };
}`


Either this can be modified so that app doesnt break, for the sake of simplicity of the code change involved, making a change in the ClientApplication was preferred.

Reply:
Not sure I understand - all top level configuration overrides should be done in `Configuration.ts`. We shouldn't be modifying the configuration anymore after that.